src/DssSpell.t.sol

```diff
@@ -2312,103 +2295,12 @@ contract DssSpellTest is DSTest, DSMath {
         vat.move(address(this), address(0x0), vat.dai(address(this)));
     }
 
-    function checkDirectIlkIntegration(
```

Comment:
I would like to keep this as it should be generic enough to reuse if/when we integrate D3M for Compound.

Reply:
Done.